openpype/tools/workfiles/window.py

```diff
@@ -29,11 +30,7 @@
 
 def file_size_to_string(file_size):
     size = 0
-    size_ending_mapping = {
-        "KB": 1024 ** 1,
-        "MB": 1024 ** 2,
-        "GB": 1024 ** 3
-    }
+    size_ending_mapping = {"KB": 1024**1, "MB": 1024**2, "GB": 1024**3}
```

Comment:
I ran `black` on the document and it automatically reformat some of these, I can revert it back

Reply:
Reverted so it only affects the lines I touched